src/ripple/app/peers/UniqueNodeList.cpp

```diff
@@ -813,8 +823,8 @@ class UniqueNodeListImp
     //
     void scoreCompute ()
     {
-        strIndex                umPulicIdx;     // Map of public key to index.
-        strIndex                umDomainIdx;    // Map of domain to index.
+        hash_map<std::string, int> umPulicIdx;     // Map of public key to index.
```

Comment:
Ultra-minor nit (and not your change): I think `umPulicIdx` is supposed to be `umPublicIdx`.